app/src/main/java/io/github/sustainow/presentation/ui/components/MultiFieldQuestionCard.kt

```diff
@@ -0,0 +1,490 @@
+                                    Box(
+                                        modifier = Modifier
+                                            .border(
+                                                1.dp,
+                                                MaterialTheme.colorScheme.outline,
+                                                MaterialTheme.shapes.small
+                                            ),
+                                        contentAlignment = Alignment.Center
+                                    ) {
+                                        innerTextField()
+                                    }
+                                },
+                                value = item.value.toString(),
+                                onValueChange = { newText ->
+                                    items = items.copy(
+                                        alternatives = items.alternatives.mapIndexed { i, alt ->
+                                            if (i == index) alt.copy(
+                                                value = newText.toFloat() ?: 0f
+                                            ) else alt
+                                        }.toMutableList()
+                                    )
+                                }
+                            )
+                        }
+                    }
+
+                    Spacer(modifier = Modifier.height(10.dp))
+                }
+            }
+
+            Box(
+                modifier = Modifier
+                    .height(48.dp)
+                    .width(48.dp)
+                    .background(MaterialTheme.colorScheme.secondaryContainer, shape = CircleShape)
+                    .padding(vertical = 10.dp)
+                    .clickable {
+                        if(isRealConsumption) {
+                            items = items.copy(
+                                alternatives = items.alternatives
+                                    .toMutableList()
+                                    .apply {
+                                        add(
+                                            QuestionAlternative(
+                                                area = "carbon",
+                                                text = "",
+                                                value = 0f,
+                                                timePeriod = Duration.ZERO,
+                                                unit = "bags"
+                                            )
+                                        )
+                                    }
+                            )
+                        }
+                    },
+                contentAlignment = Alignment.Center,
+            ) {
+                Icon(
+                    Icons.Outlined.Add,
+                    tint = MaterialTheme.colorScheme.secondary,
+                    contentDescription = null
+                )
+            }
+        }
+    }
+}
```

Comment:
Botão pode ser um IconButton simples, só precisa ser renderizado no menu expandido e quando questionAlternative for substituída por FormAnswer. A unidade de medida pode ser recebida como parâmetro do componente para evitar valores hardcoded